apps/server/src/utils/generateRaceText.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+export const generateRaceText = async (): Promise<string> => {
```

Comment:
shouldn't this be parameterized to include paragraph count and other attributes? of text

Reply:
Currently, we only need 1 paragraph for race. we can expand it later.